hazelcast/src/main/java/com/hazelcast/internal/util/StringUtil.java

```diff
@@ -57,28 +57,11 @@ public final class StringUtil {
     public static final Pattern VERSION_PATTERN
             = Pattern.compile("^(\\d+)\\.(\\d+)(\\.(\\d+))?(-\\w+(?:-\\d+)?)?(-SNAPSHOT)?$");
 
-    /**
-     * Empty String.
-     */
-    public static final String EMPTY_STRING = "";
```

Comment:
Why is this removed?

Reply:
It is dead code.

Reply:
It is not used

Reply:
> It is dead code.

> It is not used

I'm pretty sure I'm actually using it in a branch, but I guess I'm the only one!